src/main/kotlin/dk/cachet/carp/webservices/data/service/IDataStreamService.kt

```diff
@@ -1,7 +1,10 @@
 package dk.cachet.carp.webservices.data.service
 
+import dk.cachet.carp.common.application.UUID
 import dk.cachet.carp.webservices.data.domain.DataStreamSequence
+import kotlinx.datetime.Instant
 
 interface IDataStreamService {
```

Comment:
I can see that you have got rid of the I prefix in DataPointService, can you also do that here?

Reply:
upon renaming to DataPointService, there's conflict in CoreDataStreamServiceunreal

Could not autowire. There is more than one bean of 'DataStreamService' type.
Beans:
coreDataStreamService   (CoreDataStreamService.kt) dataStreamServiceImpl   (DataStreamServiceImpl.kt)

Reply:
Yeah sure, I think one of us can just sit down when we have time and do the formatting and fix the naming conventions in one go, we kind of have other things to attend to.